CMakeLists.txt

```diff
@@ -2780,6 +2819,28 @@ if(ENABLE_PLUGIN_SYSTEMD_JOURNAL)
         endif()
 endif()
 
+if(ENABLE_PLUGIN_SYSTEMD_UNITS)
+        if(NOT SYSTEMD_FOUND)
+                message(FATAL_ERROR "Systemd units plugin requires systemd, but systemd was not found.")
+        endif()
```

Comment:
This actually needs to depend not on Systemd itself being found, but the systemd-dbus stuff specifically, just like the plugin originally did. Very old versions of systemd that we unfortunately still need to support (because of AL 2 and CentOS 7) did not provide that functionality, so the plugin fails to build against them.